dolphin/db/api.py

```diff
@@ -282,3 +282,23 @@ def access_info_get_all(context, marker=None, limit=None, sort_keys=None,
 def is_orm_value(obj):
     """Check if object is an ORM field."""
     return IMPL.is_orm_value(obj)
+
+
+def alert_source_create(context, values):
+    """Create an alert source."""
+    return IMPL.alert_source_create(context, values)
+
+
+def alert_source_update(context, storage_id, values):
+    """Update an alert source."""
+    return IMPL.alert_source_update(context, storage_id, values)
+
+
+def alert_source_get(context, storage_id):
```

Comment:
We need to fetch alert source when we want to get storage id from source ip.  So we need one alert source get all api with filter

Reply:
This PR is for the user side, that will be done in the next PR, otherwise PR will became too big.